.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+
+[*.json]
+indent_size = 2
+
+[*.{markdown,md}]
+indent_size = 4
+max_line_length = 80
+
+[*.py]
+indent_size = 4
+max_line_legth = 79
```

Comment:
I think this is going to be the one sticking point here in this file. I know that PEP 8 recommends a line length of 79, but I think there is good reason not to follow that nowadays. The two main reasons I would argue against it in this particular case are current convention in the project, and `black` which I think we should aim to adopt. Currently, [flake8 is configured](https://github.com/Sceptre/sceptre/blob/8a805555886e160b5325e6260949e7e9c9fa9e89/.flake8#L15) to use a line length of 120. I'm okay with using that, but nowadays I tend to use the default from `black`, [which is 88](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length).